framework/include/cppmicroservices/BundleContext.h

```diff
@@ -1070,11 +1070,9 @@ class US_Framework_EXPORT BundleContext
  * which may contain several bundles, i. e. acts as a bundle library.
  *
  * @remarks
- *     If the bundleManifest is passed in, it is installed. When the bundle is Started, the
- *     BundleManifest is NOT read from the file (as it has already been installed). In the event
- *     that the injected bundle manifest does NOT match the manifest in the bundle's file, the
- *     behavior of the system is undefined. That is, the content of the injected manifest and the
- *     manifest on disk are expected to be the same and are not compared.
+ *     If the bundleManifest is passed in, it is installed. In the event that the injected bundle
+ *     manifest does NOT match the manifest in the bundle's file, the behavior of the system is
+ *     undefined.
```

Comment:
How about this instead:

`
If the provided bundleManifest does not match the manifest embedded in the bundle's file the behavior of that bundle in CppMicroServices is undefined.
`

Reply:
Done.